### PR TITLE
Fix scroll down styling

### LINKED ISSUE #56: Fix scroll down
Fix the scroll down button to make it look exactly like mock.
https://mirr.io/YKK1w

### PR DESCRIPTION
**Pull request by anshumanv**
Fixes #56 

For reference.
image

### CODE REVIEW

File: src/components/eventList/styles.css [hunk 1]
```diff
@@ -8,14 +8,12 @@
 
 .start-button {
 	margin-top: 30px;
-	font-size: 24px;
-	font-family: 'Slabo 27px', serif;
+	font-size: 12px;
+	font-family: 'Sans', serif;
```
Comment (Monal5031) [inline src/components/eventList/styles.css]: Font family is Vidaloka

**Comment by anshumanv** [comment]
Refer mocks man -_-

**Comment by Monal5031** [comment]
@anshumanv My bad. The font is Roboto mono

**Comment by anshumanv** [comment]
Okay thanks, changing.

**Pull request merged**